xml/installation-installation-gui_installer.xml

```diff
@@ -410,32 +786,21 @@
    </para>
   </listitem>
  </itemizedlist>
- <note>
-  <para>
-   Because the GUI installer is disabled after deployment is complete, whether
-   you installed your cloud via the GUI or via the command line, if you want to
-   re-enable the GUI later but are not re-installing from scratch, run these
-   commands:
-  </para>
-<screen>sudo a2ensite dayzero-apache.conf
-sudo systemctl start dayzero.service</screen>
- </note>
- </section>
+  </section>
 <section>
-<title>Final Steps</title>
- <itemizedlist>
-  <listitem>
-   <para>
-    The public Horizon Dashboard connection is over TLS, thus please use
-    HTTPS:// to navigate to Horizon with the IP address the installer presents
-    once install is complete.
-   </para>
-  </listitem>
-  <listitem>
-   <para>
-    Visit the <xref linkend="ceph_overview"/> for Ceph post-install steps.
-   </para>
-  </listitem>
- </itemizedlist>
+<title>Use the Install UI securely</title>
+ <para>
+  The Install UI is a web-based application that runs on the &lcm; node and is
+  implemented in ReactJS and Python. This application runs behind Apache 2.0 on
+  port 3000. Browser refresh is not necessary if you are using the browser
+  connected directly to the Install UI.
+ </para>
+ <para>
+  You should access the Install UI through Apache. An example URL looks like
+  this: http://&lt;lifecyclemanager-host-ip&gt;:3000.
```

Comment:
Put the URL in a `<literal>` and use `<replaceable>`: `<literal>http://<replaceable>LIFE_CYCLE_MANAGER_HOST_IP</replaceable>:3000</literal>`

Also, is that port number correct? If I understand Jeremy correctly, it should be 8080 instead..?

Reply:
this is complicated... if the user is accessing the installation UI directly on the server that its running on, then they will use port 3000 ... if they're using an ssh tunnel to map port 8080 on their localhost to port 3000 on the lifecycle manager, then they'll need to use port 8080... I believe in this context that port 3000 is correct, but this does make me think that we need to clarify this a bit

Reply:
Ah, ok -- since I only looked at the sections that changed, there is a good chance that I missed that. Thanks for the explainer!